wandb/integration/yolov8/yolov8.py

```diff
@@ -0,0 +1,266 @@
+from typing import List, Optional
+
+from ultralytics.yolo.engine.model import YOLO
+from ultralytics.yolo.engine.trainer import BaseTrainer
+from ultralytics.yolo.utils import RANK
+from ultralytics.yolo.utils.torch_utils import get_flops, get_num_params
+from ultralytics.yolo.v8.classify.train import ClassificationTrainer
+
+import wandb
+from wandb.sdk.lib import telemetry
+
+
+class WandbCallback:
+    """
+    A YOLO model wrapper that tracks metrics, and logs models to Weights & Biases.
+    Usage:
+    ```python
+    from wandb.yolov8 import WandbLogger
+    model = YOLO("yolov8n.pt")
+    with WandbLogger(model,) as wb_model:
+        wb_model.train(data="coco128.yaml", epochs=3, imgsz=640,)
+    ```
+    """
+
+    def __init__(
+        self,
+        yolo: YOLO,
+        run_name: Optional[str] = None,
+        project: Optional[str] = None,
+        tags: Optional[List[str]] = None,
+        resume: Optional[str] = "allow",
```

Comment:
Is this the correct desired default?

In general, resuming runs should be an exception, not the common path.   I have noticed this show up in other integrations so I wanted to understand the motivation a bit more before adding it to to another integration.

Reply:
:shrug: I just followed the other integrations here. I will defer to your call on this and default to `None` as in the `wandb.init`